src/common/syncjournaldb.cpp

```diff
@@ -950,7 +950,7 @@ Result<void, QString> SyncJournalDb::setFileRecord(const SyncJournalFileRecord &
     if (!_etagStorageFilter.isEmpty()) {
         // If we are a directory that should not be read from db next time, don't write the etag
         QByteArray prefix = record._path + "/";
-        foreach (const QByteArray &it, _etagStorageFilter) {
+        for (const QByteArray &it : _etagStorageFilter) {
```

Comment:
I think while we are at it we might as well convert all of these to `const auto`